### ISSUE TITLE
_init called twice on doubled scene

### ISSUE THREAD

**Issue by bitwes** [bitwes/Gut#297]
I'm not sure how it's happening, but I created a button that had calls to `connect` in `_init`.  When creating a double of a scene with that script `_init` is being called twice.  This generated "already connected" errors which is how I found it.

**Comment by bitwes** [comment]
Maybe the double is including a call to `._init()`  in the `_init()` method.  I believe `_init()` is one of the methods that is called on the super class automatically.

**Comment by bitwes** [comment]
`res://addons/gut/utils.gd` now has a list of all methods that should not be stubbed to call super.  Stubbing these methods to call super has no affect on a double.

The list as of now is:
```
var non_super_methods = [
	"_init",
	"_ready",
	"_notification",
	"_enter_world",
	"_exit_world",
	"_process",
	"_physics_process",
	"_exit_tree",
	"_gui_input	",
]
```

**Comment by bitwes** [comment]
Released in v7.3.0.
